Crash when setting attendance for a staff that has not been synced

```2019-07-06 13:24:30.792 26792-26801/np.com.naxa.staffattendance W/SQLiteConnectionPool: A SQLiteConnection object for database '/data/user/0/np.com.naxa.staffattendance/databases/staffManager2' was leaked!  Please fix your application to end transactions in progress properly and to close the database when it is no longer needed.
2019-07-06 13:24:41.351 26792-26792/np.com.naxa.staffattendance D/AndroidRuntime: Shutting down VM
2019-07-06 13:24:41.621 26792-26792/np.com.naxa.staffattendance E/AndroidRuntime: FATAL EXCEPTION: main
    Process: np.com.naxa.staffattendance, PID: 26792
    java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=1, result=-1, data=Intent { act=org.idpass.mobile.intents.IDENTIFY_RESULT (has extras) }} to activity {np.com.naxa.staffattendance/np.com.naxa.staffattendance.attendence.v2.AttedanceActivity}: java.lang.NumberFormatException: For input string: "1562390658762"
        at android.app.ActivityThread.deliverResults(ActivityThread.java:4090)
        at android.app.ActivityThread.handleSendResult(ActivityThread.java:4133)
        at android.app.ActivityThread.-wrap20(ActivityThread.java)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1534)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:154)
        at android.app.ActivityThread.main(ActivityThread.java:6121)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
     Caused by: java.lang.NumberFormatException: For input string: "1562390658762"
        at java.lang.Integer.parseInt(Integer.java:524)
        at java.lang.Integer.parseInt(Integer.java:556)
        at np.com.naxa.staffattendance.attendence.v2.AttedanceActivity.saveAttendance(AttedanceActivity.kt:129)
        at np.com.naxa.staffattendance.attendence.v2.AttedanceActivity.onActivityResult(AttedanceActivity.kt:114)
        at android.app.Activity.dispatchActivityResult(Activity.java:6976)
        at android.app.ActivityThread.deliverResults(ActivityThread.java:4086)
        at android.app.ActivityThread.handleSendResult(ActivityThread.java:4133) 
        at android.app.ActivityThread.-wrap20(ActivityThread.java) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1534) 
        at android.os.Handler.dispatchMessage(Handler.java:102) 
        at android.os.Looper.loop(Looper.java:154) 
        at android.app.ActivityThread.main(ActivityThread.java:6121) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779) 
2019-07-06 13:24:42.950 26792-26822/np.com.naxa.staffattendance I/CrashlyticsCore: Crashlytics report upload complete: 5D2023F70160-0001-68A8-65388426D460```

--- a/app/src/main/java/np/com/naxa/staffattendance/attendence/v2/AttedanceActivity.kt
+++ b/app/src/main/java/np/com/naxa/staffattendance/attendence/v2/AttedanceActivity.kt
@@ -126,7 +126,7 @@ class AttedanceActivity : BaseActivity(), StaffListAdapter.OnStaffItemClickListe
         val attendanceResponse = AttendanceResponse()
         attendanceResponse.setAttendanceDate(loadedDate)
         attendanceResponse.setStaffs(listOf(staff.id))
-        attendanceResponse.idPassProofs = Gson().toJson(hashMapOf(staff.id.toInt() to signedAction))
+        attendanceResponse.idPassProofs = Gson().toJson(hashMapOf(staff.id.trim().toBigInteger() to signedAction))
         attendanceResponse.dataSyncStatus = AttendanceDao.SyncStatus.FINALIZED
         AttedanceLocalSource.instance.updateAttendance(loadedDate, attendanceResponse, staff.teamID)
 

--- a/app/src/main/java/np/com/naxa/staffattendance/database/AttendanceDao.java
+++ b/app/src/main/java/np/com/naxa/staffattendance/database/AttendanceDao.java
@@ -213,12 +213,12 @@ public class AttendanceDao {
             JSONArray jsonArray = new JSONObject(attedance.getIDPassProofs()).names();
             ArrayList<Integer> ids = new ArrayList<>();
             for (int n = 0; n < jsonArray.length(); n++) {
-                String id = jsonArray.getString(n);
+                String id = jsonArray.getString(n).replace("\"", "").trim();
                 ids.add(Integer.valueOf(id));
             }
 
             contentValues.put(DatabaseHelper.KEY_STAFFS_IDS, ids.toString());
-        } catch (JSONException e) {
+        } catch (NumberFormatException | JSONException e) {
             Timber.e(e);
         }
 
@@ -277,7 +277,7 @@ public class AttendanceDao {
             attendanceResponse.setAttendanceDate(attendanceDate);
 
 
-            String idpassproofs = DatabaseHelper.getStringFromCursor(cursor,DatabaseHelper.KEY_ID_PASS_PROOFS);
+            String idpassproofs = DatabaseHelper.getStringFromCursor(cursor, DatabaseHelper.KEY_ID_PASS_PROOFS);
             try {
                 String arrayString = new JSONObject(idpassproofs).names().toString();
                 attendanceResponse.setStaffs(convertStaffIdsToList(arrayString));

